### PR TITLE
Allow passing user agent as a CLI argument

### LINKED ISSUE #57: Feature: Change user agent
It would be greate to specify a user agent from the CLI a give it to reqwest

### PR DESCRIPTION
**Pull request by creativcoder**
Fixes #57 

Also ran `cargo fmt` on it.

### CODE REVIEW

File: src/downloader.rs [hunk 1]
```diff
@@ -6,18 +6,18 @@ use crate::warn;
...
 }
 
 impl Downloader {
     /// Create a new Downloader
-    pub fn new(tries: usize) -> Downloader {
+    pub fn new(tries: usize, user_agent: &str) -> Downloader {
         Downloader {
             client: reqwest::blocking::ClientBuilder::new()
-                .cookie_store(true)
+                .cookie_store(true).user_agent(user_agent)
```
Comment (CohenArthur) [inline src/downloader.rs:17]:
```suggestion
                .cookie_store(true)
                .user_agent(user_agent)
```

For consistency

**Comment by CohenArthur** [comment]
Thanks a lot ! 

Why did you comment a test ?

File: src/downloader.rs (hunk 1 above)
Comment (creativcoder) [reply src/downloader.rs:17]: Added this change

**Comment by creativcoder** [comment]
> Why did you comment a test ?

Ah my bad. Reverted

File: src/args.rs [hunk 2]
```diff
@@ -37,6 +37,10 @@ pub struct Args {
     /// The max seconds of delay between downloads
     #[structopt(long, default_value = "0", help="Generate an extra random delay between downloads, from 0 to this number. This is added to the base delay seconds")]
     pub random_range: u64,
+
+    /// User agent to be used to send requests
+    #[structopt(long, default_value = "suckit", help="User agent to be used for sending requests")]
```
Comment (Skallwar) [inline src/args.rs:42]:
```suggestion
    #[structopt(short, long, default_value = "suckit", help="User agent to be used for sending requests")]
```

File: src/args.rs [hunk 3]
```diff
@@ -37,6 +37,10 @@ pub struct Args {
     /// The max seconds of delay between downloads
     #[structopt(long, default_value = "0", help="Generate an extra random delay between downloads, from 0 to this number. This is added to the base delay seconds")]
     pub random_range: u64,
+
+    /// User agent to be used to send requests
+    #[structopt(short, default_value = "suckit", help="User agent to be used for sending requests")]
```
Comment (Skallwar) [inline src/args.rs:42]:
```suggestion
    #[structopt(short, long, default_value = "suckit", help="User agent to be used for sending requests")]
```

**Pull request merged**